### PR TITLE
This ones for you Dan, I hope it helps.

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

**Comment by elbing** [comment]
LGTM

File: aarch64/bin/auth/disable [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/rc
```
Comment (dancrossnyc) [inline aarch64/bin/auth/disable]: Hmm; we need this for each arch? Surely this is a candidate for /rc/bin/auth and a 'bind -a /rc/bin/auth /bin/auth' ?

**Comment by dancrossnyc** [comment]
Ron beat me to it. :-)

**Comment by rminnich** [comment]
So dan the clear.json script does an rm on /$ARCH/bin but then has to do a checkout to get the scripts back. It's because the scripts should not even be there.

**Comment by dancrossnyc** [comment]
Ha! Sad Trombone....

**Comment by dancrossnyc** [comment]
Hmm. Travis seems to have completed successfully, but Github hasn't picked up on that yet. Is there some way to force it to check?

**Comment by dancrossnyc** [comment]
(I think I found a way.)

**Pull request merged**